ruby/vampires.rb

```diff
@@ -0,0 +1,97 @@
+# Methods to check user input
+def check_if_integer
+  begin
+    num = Integer(gets.chomp)  
+  rescue 
+    puts 'Error reading input - please enter an integer number:'
+    retry
+  end
+
+  num
+end
+
+def yes_or_no
+  loop do
+    case gets.chomp.downcase
+    when 'yes'
+      return 'yes'
```

Comment:
@Bontgoy Since these statements are inside of a loop do, I think I actually need the return statements in this case in order to exit the loop.